pep/app/pods/application/controller.ts

```diff
@@ -96,6 +113,29 @@ export default class Application extends Controller {
     updateMatchSynonyms(isChecked: boolean) {
         this.matchSynonyms = isChecked;
     }
+
+    /**
+     * Open the about modal dialog
+     *
+     * @returns Promise<Void>
+     * @memberof PageDrawer
+     */
+    @action
+    async openAboutModal() {
+        try {
+            this.loadingBar.show();
+            const status = await this.ajax.request<ServerStatus>('Session/Status');
+            return this.modal.open('user/about', {
+                serverInformation: status,
+                clientBuildVersion: ENV.buildVersion
+            });
+        } catch (error) {
+            this.notifications.error(error);
```

Comment:
probably doubtful the api is returning us a string we can use directly as an error message here, so I'd probably pass in `this.intl.t('serverErrors.unknown.unexpected')` to the `error()` call.